.bazelrc

```diff
@@ -1,11 +1,31 @@
-build --cxxopt -std=c++20
-build --config=cuda
 build --experimental_cc_implementation_deps
 
 build --copt -fPIC
 
-build:portable_glibc --copt -fPIC
 build:portable_glibc --define SXT_PORTABLE_GLIBC=1
+build:portable_glibc --action_env=BAZEL_LINKLIBS='-l%:libstdc++.a'
+build:portable_glibc --action_env=BAZEL_LINKOPTS='-static-libstdc++ -static-libgcc'
+
+build --flag_alias=enable_cuda=@rules_cuda//cuda:enable
+build --flag_alias=cuda_archs=@rules_cuda//cuda:archs
+build --flag_alias=cuda_compiler=@rules_cuda//cuda:compiler
+build --flag_alias=cuda_copts=@rules_cuda//cuda:copts
+build --flag_alias=cuda_host_copts=@rules_cuda//cuda:host_copts
+build --flag_alias=cuda_runtime=@rules_cuda//cuda:runtime
+
+build --@rules_cuda//cuda:archs=compute_70:compute_70,sm_70
+build --@rules_cuda//cuda:runtime=@local_cuda//:cuda_runtime_static
+
+build --enable_cuda=True
+
+# Use --config=clang to build with clang instead of gcc and nvcc.
```

Comment:
Do you intend to update CI in a follow-up PR to use this `--config=clang` flag?

Reply:
It's set in the .bazelrc